### PR TITLE
add more pytorch ops and symbols

### PR DESCRIPTION
**Pull request by Basicname**
Add more pytorch ops and symbols
Add op : + - * @ / nn.Linear nn.Conv2d nn.ReLU nn.Sigmoid nn.Tanh nn.Softmax nn.MSELoss nn.BCELoss nn.CrossEntropyLoss nn.ConvTranspose2d nn.MaxUnpool2d nn.BatchNorm2d nn.AvgPool2d nn.MaxPool2d nn.Dropout nn.Embedding nn.LSTM nn.AdaptiveMaxPool2d nn.AdaptiveAvgPool2d

### CODE REVIEW

**Comment by Tokisakix** [comment]
Thanks for your contribution! I check the torch ops and symbols in my pc(torch2.4.1+cu124) and it's all right!
